interfaces/tls-certificates/src/charmlibs/interfaces/tls_certificates/_version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "1.0.0.post2"
+__version__ = "1.0.0"
```

Comment:
We should either bump the minor version for the next release:
```suggestion
__version__ = "1.1.0"
```
Or drop the version change if you want to make any further changes (e.g. migrating tests) before releasing.

Reply:
Bumped.